### DISCUSSION TITLE
Discovered Overseerr by accident. Impressed

### DISCUSSION THREAD

**Discussion by chrisfoster1971** [sct/overseerr#1033] [General]
I stumbled across this project earlier today and decided to give it a bash using rancher on ubuntu server.

Worked first time, hooked up to Plex, my multiple sonarr/radarr etc.

I've a couple of queries if I may?
1. Is it possible to enable the addition of multiple Plex instances servers and their respective libraries within the settings screen, or have I misunderstood how this feature is working. I've added each and sync'd libraries but it gives the impression that only one Plex instance is sync'd at any one time, is that correct?
1. Is there any plans to support Lidarr?

Next step to setup with Traefik. Keep up the amazing work.

**Comment by sct** [comment]
You can only have one Plex server active at any time. No plans for that to change. Basically, it complicates things a ton if we want to support more than one plex server at a time.

As for Lidarr support, someday. There is a feature request here: https://github.com/sct/overseerr/issues/352 that you can follow for updates, but it will be a fairly long time before we can even begin on it. The current focus is perfecting movie/series requests.

**Comment by TheCatLady** [comment]
@chrisfoster1971 Like @sct said, you can only have one Plex server associated with an Overseerr instance.  However, there's nothing stopping you from running multiple Overseerr instances (one for each Plex server)!

**Comment by chrisfoster1971** [comment]
Thank you for your reply.
Glad it wasn't just me then. With the import plex servers picking my
instances up i thought I was doing something wrong.
It's already impressive as it is. Thank you
